### PR TITLE
Arcane Jolt sheet usage - intelligence modifier, minimum of one

### LINKED ISSUE #134: [ERLW] Artificer Arcane Jolt
Artificer feature Arcane Jolt is missing the sheet usage information

### PR DESCRIPTION
**Pull request by mamawerecat**
resolves #134

### CODE REVIEW

**Comment by kgirtxd** [comment]
Can you also double check that there are not other features in that file with the same issue?

**Comment by mamawerecat** [comment]
> Can you also double check that there are not other features in that file with the same issue?

Sure thing!

**Comment by mamawerecat** [comment]
> Can you also double check that there are not other features in that file with the same issue?

Everything else looks good, no other issues that I saw.

File: supplements/eberron-rising-from-the-last-war/class-artificer.xml [hunk 1]
```diff
@@ -824,14 +824,16 @@
 			</ul>
 			<p class="indent">You can use this energy a number of times equal to your Intelligence modifier (minimum of once), but you can do so no more than once on a turn. You regain all expended uses when you finish a long rest.</p>
 		</description>
-		<sheet>
+		<sheet alt="Arcane Jolt" usage="{{arcane jolt usage:intelligence}}/Long Rest">
```
Comment (kgirtxd) [inline supplements/eberron-rising-from-the-last-war/class-artificer.xml:827]:
```suggestion
		<sheet usage="{{arcane jolt usage:intelligence}}/Long Rest">
```
Alt is unneeded when its the same name as the feature

**Review by kgirtxd — CHANGES_REQUESTED** [review]

File: supplements/eberron-rising-from-the-last-war/class-artificer.xml [hunk 2]
```diff
@@ -824,14 +824,16 @@
 			</ul>
 			<p class="indent">You can use this energy a number of times equal to your Intelligence modifier (minimum of once), but you can do so no more than once on a turn. You regain all expended uses when you finish a long rest.</p>
 		</description>
-		<sheet>
+		<sheet usage="{{arcane jolt usage:intelligence}}/Long Rest">
```
Comment (NickVendel) [inline supplements/eberron-rising-from-the-last-war/class-artificer.xml:827]:
```suggestion
		<sheet usage="{{arcane jolt:usage}}/Long Rest">
```

File: supplements/eberron-rising-from-the-last-war/class-artificer.xml [hunk 3]
```diff
@@ -824,14 +824,16 @@
...
 			<description>When either you hit a target with a magic weapon attack or your steel defender hits a target, you can create one of the following effects:
 			The target takes an extra {{arcane jolt damage:dice:amount}}d6 force damage.
 			Choose one creature or object you can see within 30 feet of the target restoring {{arcane jolt healing:dice:amount}}d6 hit points to it.</description>
 		</sheet>
 		<rules>
 			<stat name="arcane jolt damage:dice:amount" value="2" />
 			<stat name="arcane jolt healing:dice:amount" value="2" />
+			<stat name="arcane jolt usage:intelligence" value="1" bonus="base" />
+			<stat name="arcane jolt usage:intelligence" value="intelligence:modifier" bonus="base" />
```
Comment (NickVendel) [inline supplements/eberron-rising-from-the-last-war/class-artificer.xml:836]:
```suggestion
			<stat name="arcane jolt:usage" value="1" bonus="base" />
			<stat name="arcane jolt:usage" value="intelligence:modifier" bonus="base" />
```

File: supplements/eberron-rising-from-the-last-war/class-artificer.xml [hunk 4]
```diff
@@ -824,14 +824,16 @@
...
 		</description>
-		<sheet>
+		<sheet usage="{{arcane jolt usage:intelligence}}/Long Rest">
 			<description>When either you hit a target with a magic weapon attack or your steel defender hits a target, you can create one of the following effects:
 			The target takes an extra {{arcane jolt damage:dice:amount}}d6 force damage.
 			Choose one creature or object you can see within 30 feet of the target restoring {{arcane jolt healing:dice:amount}}d6 hit points to it.</description>
 		</sheet>
 		<rules>
 			<stat name="arcane jolt damage:dice:amount" value="2" />
 			<stat name="arcane jolt healing:dice:amount" value="2" />
```
Comment (NickVendel) [inline supplements/eberron-rising-from-the-last-war/class-artificer.xml:834]: it's usually "[feature]:[effect]:[details]", so stats before new ones are wrong too, you should change those to just be one stat as `"arcane jolt:dice:amount"` with value 2 and a bonus `"base"`. Then in Improved Defender feature, do the same thing and use value 4 with a bonus `"base"`.

Comment (mamawerecat) [reply supplements/eberron-rising-from-the-last-war/class-artificer.xml]: I _think_ I made the changes you requested. My brain is a little sluggish with a migraine

**Review by NickVendel — APPROVED** [review]
Yup, looks good.

**Comment by kgirtxd** [comment]
Absorbed by #180

**Pull request closed without merging**